src/main/java/com/rasklaad/wiremock/metrics/MetricsConfiguration.java

```diff
@@ -13,6 +13,7 @@ public class MetricsConfiguration {
     private boolean responseSendTimeEnabled = true;
 
     private Double maximumMetricExpectedValue;
+    private Double minimumMetricExpectedValue;
```

Comment:
This option is never used in `PrometheusMetricsExtension`. Right now you can define minimumMetricExpectedValue, but it won't affect anything in metrics.